Tests/PageObject/RegisterPage.py

```diff
@@ -0,0 +1,48 @@
+import faker
+
+from Tests.PageObject.BasePage import BasePage
+from Tests.PageObject.locators import RegisterPageLocators
+
+fake = faker.Faker()
+password = fake.password()
+first_name = fake.first_name()
+last_name = fake.last_name()
+
+
+class RegisterPage(BasePage):
+    def guest_can_go_to_register_page(self):
+        self.should_be_customer_form()
+
+    def should_be_customer_form(self):
+        assert self.is_element_present(*RegisterPageLocators.RIGHT_PANEL), "Right panel is not present"
+        assert self.is_element_present(*RegisterPageLocators.CUSTOMER_FORM), "Customer form is not present"
+
+    def register_new_fake_user(self):
+        self.browser.find_element(*RegisterPageLocators.FIELD_FIRST_NAME).send_keys(first_name)
```

Comment:
same here, each element should be a separate function. register_new_fake_user(self) is a helper function and should be moved away from this class